types/chrome/index.d.ts

```diff
@@ -4922,7 +4922,7 @@ declare namespace chrome.platformKeys {
     }
 
     /**
-     * This function filters from a list of client certificates the ones that are known to the platform, match request and for which the extension has permission to access the certificate and its private key. If interactive is true, the user is presented a dialog where he can select from matching certificates and grant the extension access to the certificate. The selected/filtered client certificates will be passed to callback.
+     * This function filters from a list of client certificates the ones that are known to the platform, match request and for which the extension has permission to access the certificate and its private key. If interactive is true, the user is presented a dialog where they can select from matching certificates and grant the extension access to the certificate. The selected/filtered client certificates will be passed to callback.
```

Comment:
The documentation for the Chrome APIs is copied verbatim from the official documentation pages written by Google. I am hesistant to changing that. If Google changes their documentation to use gender-neutral pronouns, those changes will propagate down to these type definitions.

Reply:
The docs seem to already have the gender-neutral wording proposed: https://developer.chrome.com/extensions/platformKeys